### PR TITLE
feat: update owners for backup-dr

### PR DESCRIPTION
**Pull request by ankushpareddy**
Adding myself and niharika as owners for backupDR terraform

### CODE REVIEW

**Review by g-awmalik — APPROVED** [review]

**Comment by g-awmalik** [comment]
@ankushpareddy - please sure owners are valid Github usernames.

File: infra/terraform/test-org/org/locals.tf [hunk 1]
```diff
@@ -806,7 +806,7 @@ locals {
       org         = "GoogleCloudPlatform"
       short_name  = "backup-dr"
       description = "Deploy Backup and DR appliances"
-      owners      = ["umeshkumhar"]
+      owners      = ["umeshkumhar", "ankushpa", "nkuravi"]
```
Comment (g-awmalik) [inline infra/terraform/test-org/org/locals.tf:809]: "ankushpa" & "nkuravi" are not valid usernames

**Review by g-awmalik — CHANGES_REQUESTED** [review]
Replace ldaps with Github usernames.

**Comment by bharathkkb** [comment]
Please join GoogleCloudPlatform github org for `ankushpareddy`

**Pull request closed without merging**